### PR TITLE
{numlib}[GCC/10.3.0] OpenBLAS v0.3.15

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.2.2004, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/c767ee5e3e7e84143804cc49362b3c71 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ebvm.novalocal - Linux centos linux 8.3.2011, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a410372c83ac8ef2bd3ce79b8e95b959 for a full test report.

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name = 'OpenBLAS'
+version = '0.3.13'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb:2]: OpenBLAS release 0.3.15 seems to be close, cfr. https://github.com/xianyi/OpenBLAS/milestone/34

(0.3.14 was essentially cancelled, due to regression in AVX512 SGEMM)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb:2]: Okay, I marked this PR as draft for now. I will update this as soon as 0.3.15 is released.

**Comment by boegel** [comment]
@seb45tian OpenBLAS 0.3.15 is out now: https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.15

**Comment by seb45tian** [comment]
> @seb45tian OpenBLAS 0.3.15 is out now: https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.15

Guess this was the wrong Sebastian :-P - Ping @SebastianAchilles

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.13-GCC-10.3.0.eb]: Done :-)

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/0c3109daa6d41af0dc6a6da7bb4be3ce for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ebvm.novalocal - Linux centos linux 8.3.2011, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/813c5f6edaf2cc90973f80469d88881a for a full test report.

**Comment by boegel** [comment]
> > @seb45tian OpenBLAS 0.3.15 is out now: https://github.com/xianyi/OpenBLAS/releases/tag/v0.3.15
> 
> Guess this was the wrong Sebastian :-P - Ping @SebastianAchilles

Oh, sorry for the confusion! *waves fist at GitHub auto-complete*

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-4xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/5afb64c9c2c382f411f49a081e892604 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/2ced40ab6af42038aed32e3b5a6c8e92 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/89e016e93005baaa4edc99285ad284df for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/dd68bcb4c0f2949592d74b8832faaaa9 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**